lib/src/realm_class.dart

```diff
@@ -278,6 +279,18 @@ class Realm implements Finalizable {
     return object;
   }
 
+  /// Ingest an [AsymmetricObject] to the [Realm].
+  ///
+  /// Ingesting is a write only operation. The ingested object will be dead immediately
```

Comment:
This seems a bit unrefined. 
This is what the docs says
>You cannot read these objects. Once created, they sync to the App Services backend and the linked Atlas database. After syncing to Atlas, Realm deletes asymmetric objects from the device. This keeps the realm file size small even with a heavy insert workload.

Could this be
>Ingesting is a write only operation. The ingested objects sync to the App Services backend and are deleted from the device. It can not be read from the `Realm`.

Reply:
Updated